core/src/main/java/info/rmapproject/core/rmapservice/impl/openrdf/ORMapAgentMgr.java

```diff
@@ -72,6 +72,16 @@
 	@org.springframework.beans.factory.annotation.Value("${rmapcore.adminAgentUri}")
 	private String adminAgentUri;
 	
+	private ORMapEventMgr eventMgr;
+
+	public ORMapAgentMgr(ORMapEventMgr eventMgr) {
+		if (eventMgr == null) {
+			throw new IllegalArgumentException("ORMapEventMgr must not be null.");
+		}
+
+		this.eventMgr = eventMgr;
+	}
+
```

Comment:
If add @Autowired to the constructor, I don't think you need to add anything to the spring config file. See comment on spring-rmapcore-context.xml